scripts/npm/prepublish/lib/changelog.js

```diff
@@ -1,21 +1,26 @@
-'use script';
-
 const fs = require( 'fs' );
-const path = require( 'path' );
 const promisify = require( 'promisify-node' );
 
 promisify( fs.writeFile );
 
-function updateChangelog( version ) {
-  const file = path.join( __dirname, '..', '..', '..', '..', 'CHANGELOG.md' );
-  const date = new Date().toJSON().slice( 0, 10 );
-  const latest = '## Unreleased\n\n### Added\n- \n\n### Changed\n- ' +
-               '\n\n### Removed\n- \n\n## ' + version + ' - ' + date;
+/**
+ * Updates the Unreleased seciton of the changelog to a versioned Release
+ * and adds a new Unreleased section
+ * @param {string} file The path of the file to edit
+ * @param {string} date The date to tag the Release with
+ * @param {string} version The version to tag the Release with
+ * @returns {function} The written file
+ */
+function updateChangelog( file, date, version ) {
+  const latest = '## Unreleased\n\n### Added\n-\n\n### Changed\n-' +
```

Comment:
Would this make more sense to move to the next line and make it a template string so you wouldn't need the newline characters?

Reply:
I can dig it.

Reply:
Oh, actually found an even better option. Pushing with the other fixes.

Reply:
Bahahaha, I broke this with my change, but luckily we have tests now so it was obvious I broke it 😄